docs/Verilog.md

```diff
@@ -44,12 +44,12 @@ Note: These operators are not shown in order of precedence.
 |---|---|
 | ~ | Bitwise NOT  |
 | & | Bitwise AND  |
-| | | Bitwise OR  |
+| &#124; | Bitwise OR  |
 | ^ | Bitwise XOR  |
 | ~^ or ^~  | Bitwise XNOR  |
 | ! | Logical NOT  |
 | && | Logical AND   |
-| || | Logical OR   |
+| &#124; &#124; | Logical OR   |
 | +  | Addition  |
 | -  | Subtraction  |
 | -  | 2's Complement  |
```

Comment:
why using codes only for `|` ? 
if you want to use codes, use for all, else none.

Reply:
If I dont use the code for "|", then it'll assume it as a different column altogether. Thats pretty much stated how to use "|" in the markdown guide